auth-web/src/components/auth/AddUsersForm.vue

```diff
@@ -155,7 +196,7 @@ export default class AddUsersForm extends Vue {
   private isFormValid (): boolean {
     let isValid: boolean = false
     this.users.forEach(user => {
-      if (user.username || user.password) {
+      if (user.username.trim() && user.password && CommonUtils.validatePasswordRules(user.password) && this.validateUserName(user.username)) {
```

Comment:
This is a bit of a confusing statement - maybe move the non-null validation into the validation functions for password and username to be more concise?

Reply:
sure will do

Reply:
I have added a change to the loop,
the null validation is need since we need to validate only the entered rows.. say 2 our 3 in the UI.

also, while checking that found another issue that, even if the validation failed for 1st row, if the 2nd row is valid, it will flag as valid.
added fix for that

Reply:
Looks good to me, thanks @nitheesh-aot